src/constant/enums.js

```diff
@@ -243,3 +243,106 @@ export const ActiveTabStatusLabels = [
 export function getActiveTabStatusLabel(activeTabStatus) {
     return getEnumLabel(ActiveTabStatusLabels, activeTabStatus);
 }
+
+export const SpotApprovalStatus = Object.freeze({
```

Comment:
Please keep it consistent with BE
https://github.com/parkspot-dev/maya/blob/master/core/models/struct.go#L562
Prefer using same enum name "SpotRequestStatus"